api/api/constants/moderation.py

```diff
@@ -0,0 +1,23 @@
+from django.db import models
+
+
+class DecisionAction(models.TextChoices):
+    """
+    This enumeration represents the actions that can be taken by a moderator as
+    a part of a moderation decision.
+    """
+
+    MARKED_SENSITIVE = "mark", "Marked sensitive"
+
+    DEINDEXED_COPYRIGHT = "deidx_copyright", "Deindexed (copyright)"
+    DEINDEXED_SENSITIVE = "deidx_sensitive", "Deindexed (sensitive)"
+
+    REJECTED_REPORTS = "rej", "Rejected"
+    DEDUPLICATED_REPORTS = "dedup", "De-duplicated"
+
+    REVERSED_MARK_SENSITIVE = "rev_mark", "Reversed deindex"
+    REVERSED_DEINDEX = "rev_deidx", "Reversed mark sensitive"
```

Comment:
What's the reason to abbreviate the slugs? We don't need to save space in these fields and it will make it much harder to remember how to query them from PGCLI. I'd much prefer descriptive, fully-spelled out slugs that are obvious and direct references to the "human-readable"/display terms. The last two also have their human-readable strings swapped.

```suggestion
    MARKED_SENSITIVE = "marked_sensitive", "Marked sensitive"

    DEINDEXED_COPYRIGHT = "deindexed_copyright", "Deindexed (copyright)"
    DEINDEXED_SENSITIVE = "deindexed_sensitive", "Deindexed (sensitive)"

    REJECTED_REPORTS = "rejected", "Rejected"
    DEDUPLICATED_REPORTS = "deduplicated", "De-duplicated"

    REVERSED_MARK_SENSITIVE = "reversed_marked_sensitive", "Reversed mark sensitive"
    REVERSED_DEINDEX = "reversed_deindexed", "Reversed deindex"
```

Reply:
There wasn't any real reason, I've switched to using full names in e32bfa0.